h/static/styles/components/_flash-messages.scss

```diff
@@ -0,0 +1,31 @@
+@use "core/color";
+@use "core/typography";
+
+.flash-message {
+  @include typography.font-big;
+
+  border-radius: 3px;
```

Comment:
Same border-radius as used by the form fields on the same page where these flash messages appear. Various different border radius's are used in h so I chose to be consistent with this one. They're all just hardcoded and duplicated throughout h, no variables seem to be used.